cmd/contour/serve.go

```diff
@@ -1053,9 +1053,10 @@ func (s *Server) setupGatewayAPI(contourConfiguration contour_api_v1alpha1.Conto
 
 		// Some features may be disabled.
 		features := map[string]struct{}{
-			"tlsroutes":  {},
-			"grpcroutes": {},
-			"tcproutes":  {},
+			"tlsroutes":        {},
+			"grpcroutes":       {},
+			"tcproutes":        {},
+			"backendtlspolicy": {},
```

Comment:
You'll need to add this as a valid option for the `--disable-feature` flag, defined above in this file (`serve.Flag("disable-feature", ...`)

Also, since the rest of the values are plural, let's do the same here -- `backendtlspolicies`.

Reply:
Done.